core/db_models/EEM_Form_Element.model.php

```diff
@@ -195,28 +151,12 @@ protected function __construct(Element $element, InputTypes $input_types, $timez
                                 'Whether form input is active, archived, trashed, or used as a default on new forms. Values correspond to the %s constants.',
                                 'event_espresso'
                             ),
-                            'EEM_Form_Input::STATUS_*'
-                        )
-                    ),
-                    false,
-                    Element::STATUS_ACTIVE,
-                    $element->validStatusOptions()
-                ),
-                'FIN_type'    => new EE_Enum_Text_Field(
-                    'FIN_type',
-                    esc_html(
-                        sprintf(
-                            /* translators: 1 class name */
-                            __(
-                                'Form input type. Values correspond to the %s constants.',
-                                'event_espresso'
-                            ),
-                            'EventEspresso\core\domain\entities\form\Input::TYPE_*'
```

Comment:
`FIN_type` should be a separate field because it is not actually a form attribute, rather it represents the type of element which allows us to decide how it should be rendered.

Reply:
kk we can keep separate from the others
i was being absolutely literal with my interpretation and using the HTML definitions for what are and are not input attributes. 
It will be easier to handle if that is a property directly on the input as opposed to being encapsulated within the attributes JSON.